src/js/outreach/views/verify_views.js

```diff
@@ -77,7 +77,7 @@ const VerifyCodeView = View.extend({
     const first = value.charAt(0);
     const rest = value.substring(1);
 
-    event.target.value = first ?? '';
+    event.target.value = first;
```

Comment:
I can’t think of a situation where `first` would be `null` or `undefined` here? 

`event.target` would have to be `null`/`undefined` somehow? Is that possible here? This is an event watcher function, so I would imagine it's not possible.

I can’t remember why I put that in there originally.